torch_xla/distributed/data_parallel.py

```diff
@@ -64,6 +64,9 @@ def next(self):
 
 
 class ParallelLoader(object):
+  """
```

Comment:
I wonder ... do these comments really help?

Reply:
No. They're placeholders. It's more important to identify the functions and classes that need docstrings and ensure they're built properly at this point.

Reply:
I am OK with these, but they need better comments.